emmo/.htaccess

```diff
@@ -0,0 +1,99 @@
+Options +FollowSymLinks
+
+# Rewrite engine setup
+RewriteEngine On
+
+
+# Rule 1: https://w3id.org/emmo
+#
+# If the request appears coming from a browser, redirect to the html documentation
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/
+RewriteRule ^$ https://emmo-repo.github.io/emmo.html [R=303,NE,L]
+#
+# ...otherwise redirect to inferred/squashed ontology
+RewriteRule ^emmo-inferred$ https://emmo-repo.github.io/emmo-inferred.ttl [R=303,NE,L]
+RewriteRule ^$ https://emmo-repo.github.io/emmo.ttl [R=303,NE,L]
+
+
+# Rule 2: https://w3id.org/emmo/
```

Comment:
as far as I know the trailing slash will be stripped - maybe use a explicit `latest`-version?

Reply:
Trailing slash seems not to be stripped on emmo.info/ but something more explicit than a trailing slash might be a good idea.